bodhi/tests/server/test_models.py

```diff
@@ -3258,22 +3242,6 @@ def test_comment_critpath_unapproved(self):
                               'additional karma from the community. Or, it must spend 14 days in '
                               'testing without any negative feedback')}])
 
-    @mock.patch('bodhi.server.mail.smtplib.SMTP')
-    @mock.patch.dict('bodhi.server.models.config',
-                     {'bodhi_email': 'bodhi@fp.o', 'smtp_server': 'smtp.fp.o'})
-    def test_comment_emails_maintainers(self, SMTP):
-        """comment() should send e-mails to the other maintainers."""
-        bowlofeggs = model.User(name=u'bowlofeggs', email=u'bowlofeggs@fp.o')
-        self.obj.builds[0].package.committers.append(bowlofeggs)
-
-        self.obj.comment(self.db, u'Here is a cool e-mail for you.', author=u'bowlofemail')
-
-        bodies = [c[1][2].decode('utf-8') for c in SMTP.return_value.sendmail.mock_calls]
-        self.assertTrue('lmacken' in bodies[0])
-        # In Python 2 this address is in the middle e-mail and in Python 3 it's in the last e-mail
-        self.assertTrue('bowlofeggs@fp.o' in '\n'.join(bodies))
-        self.assertTrue(all(['Here is a cool e-mail for you.' in b for b in bodies]))
```

Comment:
Heh, this test makes me think that we were supposed to do this. The problem is that nothing ever set the committers to anything on packages so this never actually happened. I guess we could fix Bodhi to actually set the committers and keep this table/code instead of deleting it. I'm not sure where would be the best place to do that though. Perhaps it would be one of the backend tasks when we create or edit an update?

What do you think? This is a feature that has obviously never worked, so I am OK with accepting this patch and dropping it. But I would also be OK with fixing it to what seems to be the intention here. Since you did the work, I'll let you decide ☺

Reply:
Hard to say. Is there any requirement for that in Bodhi?
Do you know about someone who mentioned that he didn't receive any notification?

Reply:
I don't know of any requirement to have this feature, and I've not heard from anyone saying they wanted e-mails that they weren't getting.

Reply:
If so, I think that we can remove this.